src/pages/Homepage.tsx

```diff
@@ -1,3 +1,14 @@
-import { Example } from '../components/Example/Example';
-
-export const Homepage = () => <Example>Hello world!</Example>;
+import { Welcome } from './sections/Welcome/Welcome';
+import { HowDoesItWork } from './sections/HowDoesItWork/HowDoesItWork';
+import { CurrentlyReading } from './sections/CurrentlyReading/CurrentlyReading';
+import { UpcomingEvents } from './sections/UpcomingEvents/UpcomingEvents';
+import { Testimonials } from './sections/Testimonials/Testimonials';
+export const Homepage = () => (
+  <div>
+    <Welcome />
+    <HowDoesItWork />
+    <CurrentlyReading />
+    <UpcomingEvents />
+    <Testimonials />
+  </div>
```

Comment:
It is fine for now. Let's see how big the sections will be. But this is a bit of overabstraction.